shell/platform/android/io/flutter/embedding/engine/FlutterEngineGroup.java

```diff
@@ -178,8 +197,19 @@ public void onEngineWillDestroy() {
   }
 
   @VisibleForTesting
-  /* package */ FlutterEngine createEngine(Context context) {
-    return new FlutterEngine(context);
+  /* package */ FlutterEngine createEngine(
+      Context context,
+      @NonNull PlatformViewsController platformViewsController,
+      boolean automaticallyRegisterPlugins,
+      boolean waitForRestorationData) {
+    return new FlutterEngine(
+        context,
+        null,
+        null,
+        platformViewsController,
+        null,
+        automaticallyRegisterPlugins,
+        waitForRestorationData);
```

Comment:
It would be nice if we added a comment to each parameter to indicate what each parameter is.

Reply:
Done~